### PR TITLE
Issue #6281: add url to hovercard avatar

### LINKED ISSUE #6281: Link avatars to profiles in hovercards
I always find myself clicking the profile pic of a hovercard to get to the profile page.

No biggie, just kind of a consistency thing.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/25282832-link-avatars-to-profiles-in-hovercards?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F522&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by AntonTkachov**
Fixed #6281

### CODE REVIEW

File: app/assets/javascripts/app/views/hovercard_view.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ app.views.Hovercard = app.views.Base.extend({
 
     // cache some element references
     this.avatar = this.$('.avatar');
+    this.avatar_link = this.$('a.person_avatar');
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/hovercard_view.js]: Identifier 'avatar_link' is not in camel case.

Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/hovercard_view.js]: Strings must use doublequote.

File: app/assets/javascripts/app/views/hovercard_view.js [hunk 2]
```diff
@@ -115,6 +116,7 @@ app.views.Hovercard = app.views.Base.extend({
     var self = this;
 
     this.avatar.attr('src', person.avatar);
+    this.avatar_link.attr('href', person.url);
```
Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/hovercard_view.js]: Identifier 'avatar_link' is not in camel case.

Comment (diaspora-code-review) [inline app/assets/javascripts/app/views/hovercard_view.js]: Strings must use doublequote.

**Comment by svbergerem** [comment]
Thank you for opening this pull request. As you can see there are some code style issues and it would be great if you could fix those. You can also test the code style locally so you can make sure that everything is ok before pushing your changes: https://wiki.diasporafoundation.org/Styleguide#Automatic_local_review

**Comment by AntonTkachov** [comment]
Fixed

**Comment by svbergerem** [comment]
@AntonTkachov Looks good. :) Could you please squash those commits into a single one? Then I'll merge this PR.

**Comment by AntonTkachov** [comment]
Done

**Comment by svbergerem** [comment]
Thank you! :)

**Pull request closed without merging**